source/package-manager/types.d.ts

```diff
@@ -17,19 +17,24 @@ export type PackageManagerConfig = {
 	id: string;
 
 	/**
- 	How to install packages, e.g. `["npm", ["install"]]`.
+ 	How to install packages (when there is a lockfile), e.g. `["npm", ["install"]]`.
   	*/
 	installCommand: Command;
 
+	/**
+	How to install packages (when there is no lockfile), e.g. `["npm", ["install"]]`.
+	*/
+	installCommandNoLockfile: Command;
```

Comment:
nit, but these don't need parentheses:

```suggestion
 	How to install packages when there is a lockfile, e.g. `["npm", ["install"]]`.
  	*/
	installCommand: Command;

	/**
	How to install packages when there is no lockfile, e.g. `["npm", ["install"]]`.
	*/
	installCommandNoLockfile: Command;
```